### ISSUE TITLE
Ref proptype on Menu component not found in Typescript definition

### ISSUE THREAD

**Issue by brochington** [mantinedev/mantine#429]
- What package has an issue (@mantine/core, mantine/hooks or other)? **@mantine/core**
- What package version are you using? **3.1.9**
- In which browser and operating system issue occurred? **VSCode**
- Do you have an idea how to fix the issue? - **Ref proptype needs to be added to Menu Component type.**
- Are you willing to participate in fixing this issue (it's totally fine if not) and create a pull request with the fix? Possibly

When trying to use the ref prop on Menu component, I am seeing a  type error:
```
Type '{ children: ReactNode; control: Element; controlRefProp: string; ref: MutableRefObject<null>; opened: boolean; }' is not assignable to type 'IntrinsicAttributes & MenuProps'.
  Property 'ref' does not exist on type 'IntrinsicAttributes & MenuProps'.ts(2322)
(JSX attribute) ref: React.MutableRefObject<null>
```

The actual ref added seems to be present and working, so I believe this is only a problem with the types.
<img width="986" alt="Screen Shot 2021-11-17 at 1 22 39 PM" src="https://user-images.githubusercontent.com/3392188/142285868-bf0f58b6-399c-4df4-a119-2ed4a2c0c15f.png">

**Comment by rtivital** [comment]
Thanks for reporting, we will fix this issue in 3.2

**Comment by rtivital** [comment]
Fix was released in 3.2
